sim/midas/src/main/cc/simif_f1.cc

```diff
@@ -177,17 +222,17 @@ uint32_t simif_f1_t::read(size_t addr) {
 #endif
 }
 
-ssize_t simif_f1_t::pull(size_t addr, char* data, size_t size) {
+size_t simif_f1_t::pcis_read(size_t addr, char* data, size_t size) {
 #ifdef SIMULATION_XSIM
-  return -1; // TODO
+  assert(false); // PCIS is unsupported in FPGA-level metasimulation
```

Comment:
What does hitting this assertion look like in XSIM?  Could you print a nice error message and tear down the simulation in a cleaner way?  I'm concerned that XSIM will spew all kinds of stack trace crap similar to what happens with VCS when it encounters DPI code that throws an exception.

Reply:
I'll check to ensure it is helpful.

Reply:
Yeah so it appears that XSIM is pretty badly broken. But i can change this in the short term to a print + exit.